projects/plugins/jetpack/modules/videopress/class.videopress-ajax.php

```diff
@@ -41,6 +43,118 @@ public static function init() {
 		return self::$instance;
 	}
 
+	/**
+	 * Ajax method that is used by the VideoPress player to get a token to play a video.
+	 *
+	 * This is used for both logged in and logged out users.
+	 *
+	 * @return void
+	 */
+	public function wp_ajax_videopress_get_playback_jwt() {
+		$guid = filter_input( INPUT_POST, 'guid' );
+		if ( empty( $guid ) ) {
+			wp_send_json_error( array( 'message' => __( 'need a guid', 'jetpack' ) ) );
+			return;
+		}
+
+		if ( ! $this->is_current_user_authed_for_video( $guid ) ) {
+			return;
+		}
+
+		$token = $this->request_jwt_from_wpcom( $guid );
+
+		if ( empty( $token ) ) {
+			wp_send_json_error( array( 'message' => __( 'Could not obtain a VideoPress playback JWT. Please try again later. (empty upload token)', 'jetpack' ) ) );
```

Comment:
This error message maybe contains some debug string not required for production ("empty upload token") => if we want to keep it as-is, maybe adding two translations could help avoid translating "Could not obtain a VideoPress playback JWT. Please try again later." twice

Reply:
For what its worth, having 2 identical error messages for 2 different conditions makes it hard to know what failed when looking through the logs. We have at least 1 other case in production jetpack code that has made it hard in the past to know why something failed. If we're going to remove `(empty upload token)` here, lets do **_something_** to differentiate this error message from the one that comes next that has the exact same error text.

Reply:
For now let's differentiate these